setup.cfg

```diff
@@ -18,3 +19,9 @@ select = B,C,E,F,W,T4,B9
 # https://github.com/myint/rstcheck#configuration-file
 # All the "auto" directives below are part of `sphinx.ext.autodoc`, and for some reason rstcheck doesn't know that.
 ignore_directives=automodule,autoclass,autoexception,autofunction,autodecorator,autodata,automethod,autoattribute
+
+[tool:pytest]
+addopts = -s -v
+
+[coverage:run]
+branch = True
```

Comment:
Consolidated tool setups in `setup.cfg` for most of the dev tools we use, and remove other config files.